familias/serializers.py

```diff
@@ -0,0 +1,208 @@
+from rest_framework import serializers
+
+from estudios_socioeconomicos.utils import save_foreign_relashionship
+
+from .models import Familia, Comentario, Integrante, Alumno, Tutor
+
+
+class ComentarioSerializer(serializers.ModelSerializer):
+    """ Serializer to represent a .models.Comentario instance
+        through a REST endpoint for the offline application
+        to submit information.
+    """
+    class Meta:
+        model = Comentario
+        fields = ('id', 'fecha', 'texto')
+
+    def create(self, family):
+        """ This function overides the default behaviour for creating
+            an object through a serializer.
+
+            This serializer dependes on a .models.Familia
+            object instance. Since we are dealing with
+            nested created objects, the familia must be
+            created first and passed as parameter to the
+            created function.
+        """
+        self.validated_data['familia'] = family
+        return Comentario.objects.create(**self.validated_data)
```

Comment:
you might want to split , this return and creation of a new familia object, in two lines in order to make it obvious that you are returning the newly created instance of family. Not obligatory for this code review to be approved.